modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -140,10 +149,94 @@ function widget( $args, $instance ) {
 			do_action( 'jetpack_search_render_filters_widget_title', $title, $args['before_title'], $args['after_title'] );
 		}
 
+		// we need to dynamically inject the sort field into the search box when the search box is enabled, and display
+		// it separately when it's not.
 		if ( ! empty( $instance['search_box_enabled'] ) ) {
 			$this->render_widget_search_form( $instance );
 		}
 
+		$default_sort = isset( $instance['sort'] ) ? $instance['sort'] : self::DEFAULT_SORT;
+		list( $orderby, $order ) = $this->sorting_to_wp_query_param( $default_sort );
+		$current_sort = "{$orderby}|{$order}";
+
+		if ( ! empty( $instance['search_box_enabled'] ) && ! empty( $instance['user_sort_enabled'] ) ) {
+			?>
+			<label class="jetpack-search-sort-wrapper">
```

Comment:
Could we add a break or something before this. At least in 2016 theme the sort box and search box end up right on top of each other. Could also deal with this in a separate PR.